### ISSUE TITLE
[bug]: token_ids has shape torch.Size([79]) - expected [77]

### ISSUE THREAD

**Issue by i3oc9i** [invoke-ai/InvokeAI#2362]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### VRAM

128GB

### What happened?

Invoke raise an exception when I use this prompt

analog style, extremely detailed photo of a victorian woman, gorgeous bluish dress, keeping flowers in the hand, (skin details, high detailed texture)+++, (full body)++, flowing long hair,  (inside a victorian room, close to a window, light comes through the window)++, rim light, (photorealistic)+, film grain, pale colours, analog, 8k hdr, ultra realistic, film photography, dslr

### Screenshots

```
token_ids has shape torch.Size([79]) - expected [77]


Traceback (most recent call last):
  File "/Users/ivano/Code/Ai/invokeai/backend/invoke_ai_web_server.py", line 1216, in generate_images
    self.generate.prompt2image(
  File "/Users/ivano/Code/Ai/invokeai/ldm/generate.py", line 469, in prompt2image
    uc, c, extra_conditioning_info = get_uc_and_c_and_ec(
  File "/Users/ivano/Code/Ai/invokeai/ldm/invoke/conditioning.py", line 30, in get_uc_and_c_and_ec
    conditioning = _get_conditioning_for_prompt(prompt, negative_prompt, model, log_tokens)
  File "/Users/ivano/Code/Ai/invokeai/ldm/invoke/conditioning.py", line 109, in _get_conditioning_for_prompt
    conditioning, _ = _get_embeddings_and_tokens_for_prompt(model,
  File "/Users/ivano/Code/Ai/invokeai/ldm/invoke/conditioning.py", line 237, in _get_embeddings_and_tokens_for_prompt
    embeddings, tokens = model.get_learned_conditioning([fragments], return_tokens=True, fragment_weights=[weights])
  File "/Users/ivano/Code/Ai/invokeai/ldm/models/diffusion/ddpm.py", line 837, in get_learned_conditioning
    c = self.cond_stage_model.encode(
  File "/Users/ivano/Code/Ai/invokeai/ldm/modules/encoders/modules.py", line 466, in encode
    return self(text, **kwargs)
  File "/Users/ivano/Code/Ai/invokeai/.venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/ivano/Code/Ai/invokeai/ldm/modules/encoders/modules.py", line 525, in forward
    base_embedding = self.build_weighted_embedding_tensor(tokens, per_token_weights, **kwargs)
  File "/Users/ivano/Code/Ai/invokeai/ldm/modules/encoders/modules.py", line 690, in build_weighted_embedding_tensor
    raise ValueError(f"token_ids has shape {token_ids.shape} - expected [{self.max_length}]")
ValueError: token_ids has shape torch.Size([79]) - expected [77]
```

### Additional context

commit f232068ab89bd80e4f5f3133dcdb62ea78f1d0f7

It seems that **commas** are counted as tokens, because when I reduce the number of commas, the prompt works

**Comment by i3oc9i** [comment]
This seems a regression  to me, indeed  I tried out old prompts ran with previous versions of invokeAi, and now we are getting this exception....

**Comment by i3oc9i** [comment]
seem solved by PR #2382 as I tested on my Mac

**Comment by i3oc9i** [comment]
#2382 is merged in the main now
